### ISSUE TITLE
作者你好，发现图像分类任务中models里没有QMF模型的代码

### ISSUE THREAD

**Issue by 18805459971** [QingyangZhang/QMF#6]
作者你好，发现图像分类任务中models里没有QMF模型的代码

**Comment by 18805459971** [comment]
if self.args.df:
            txt_img_out = (txt_out * txt_conf.detach() + img_out * img_conf.detach()) 
这句话是否就是决定是否是QMF还是late_fusion模型的关键啊

**Comment by QingyangZhang** [comment]
是的，这里通过对模型预测进行加权求和实现动态融合
